src/faucet/views.rs

```diff
@@ -13,6 +13,7 @@ use leptos_use::*;
 use crate::faucet::controller::FaucetController;
 use crate::faucet::utils::format_balance;
 
+const FAUCET_TOPUP_REQ_URL: &str = "https://github.com/ChainSafe/forest-explorer/discussions/134";
```

Comment:
Could we make it configurable? I can add the link to this repo's environment variables.

Reply:
Otherwise, if anyone would like to deploy the faucet themselves, they'd end up with the same link.

Reply:
And once done, let's add an entry to the changelog.

Reply:
@LesnyRumcajs yes we can make it configurable, can you please add `FAUCET_TOPUP_REQ_URL`